software/v1-enlight/dia_firmware.cpp

```diff
@@ -206,7 +207,6 @@ int CreateSession() {
 int EndSession() {
     if(_AuthorizedSessionID != ""){
         int answer = network->EndSession(_AuthorizedSessionID);
-        std::cout<<"\nEndSession: "<<_AuthorizedSessionID<<"\nanswer:"<<answer;
```

Comment:
\n always MUST be in the end of the line, neve at the start
also never use \n, use std::endl()